src/test/java/org/datavyu/plugins/ffmpeg/FfmpegMediaPlayerTest.java

```diff
@@ -0,0 +1,54 @@
+package org.datavyu.plugins.ffmpeg;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import javax.swing.*;
+
+public class FfmpegMediaPlayerTest extends MediaPlayerTest {
+
+    @DataProvider(name = "ffmpegPlayers")
+    public Object[][] createPlayerWithMedia() {
+        // TODO(fraudies): Work with all video files
+        MediaInformation mediaInformation = MEDIA.stream().findFirst().get();
```

Comment:
What if we have multiple videos to test with, this will assume that you will create a data provider for each test class and before each test you are creating the data provider. A best practice would be to have all your providers in one class (to ease the maintenance and updates) and initialize each test class with a factory method where you will persist the MediaPlayerSync and MediaInformation objects to be used in your entire test suite, no need to tag all your test cases with @Test(dataProvider = "ffmpegPlayers") and you could remove the "MediaPlayerSync player, MediaInformation mediaInformation"  arguments as well. Putting all your data providers in a resource class with factory methods in your test class is more efficient and flexible.